src/dlgprefreplaygain.cpp

```diff
@@ -108,8 +108,7 @@ void DlgPrefReplayGain::slotUpdateReplayGainBoost() {
 
 void DlgPrefReplayGain::setLabelCurrentReplayGainBoost(int value) {
     LabelCurrentReplayGainBoost->setText(
-            QString(tr("%1 dB (average %2 dB)")).arg(
-                  QString().sprintf("%+d", value), QString::number(value - 14)));
+            QString(tr("Adjust by %1 dB")).arg(QString().sprintf("%+d", value)));
```

Comment:
For me the 14 dB average is the most significant value. If you prefer to have only one value, we should switch to use only the average. But IMHO the original version was more clear. Probably it is just a wording issue.